### PR TITLE
Create -r3 release. patchelf change-rpath applied

### LINKED ISSUE #2: discord-rpc rpath is changed in the ebuild
The ebuild changes the `rpath` of the discord-rpc binary (which is used to scan for libraries in the folder discord-rpc is in) using
`chrpath -d "squashfs-root/public/discord-rpc/discord-rpc" || die "Patching Library Failed"`

This completely breaks the discord-rpc. I would suggest 2 potential fixes:-

1. Getting rid of the `chrpath`
2. Writing a separate ebuild for libdiscord-rpc and installing the library to `/usr/lib` or `/lib` (needs to be tested, although should work)

### PR DESCRIPTION
**Pull request by natimerry**
# 2.3.8-r3 Release

- Removed `chrpath`, switched to [`patchelf`](https://man.archlinux.org/man/community/patchelf/patchelf.1.en) to patch discord-rpc's `rpath` during preparation phase. 
- Uses `fperms` to set `discord-rpc` binary permissions as `doexe/exeinto` does not seem to work. 
- Added `dev-util/patchelf` to build depends. Removed older `app-admin/chrpath` 

Follow up to Issue #2 and PR #3  

Closes #2

### CODE REVIEW

**Comment by Schievel1** [comment]
Thanks for that, I appreciate your efforts. And thank you for not just merging this in this time 😄 

I will have a look later and test this out. Maybe also give the doexe issue a try. fperms is fine actually, but now I want to know why this doesn't work.

File: games-misc/an-anime-game-launcher-bin/an-anime-game-launcher-bin-2.3.8-r3.ebuild [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	exeinto "/usr/lib/${PN}"
+	doexe "squashfs-root/an-anime-game-launcher"
+	doins -r "squashfs-root/public"
+	insinto "/usr/share/pixmaps"
+	doins "${PN}.png"
+	exeinto "/usr/bin"
+	doexe "${PN}"
+	insinto "/usr/share/applications/"
+	doins "${PN}.desktop"
+	fperms 755 "/usr/lib/${PN}/public/discord-rpc/discord-rpc"
```
Comment (Schievel1) [inline games-misc/an-anime-game-launcher-bin/an-anime-game-launcher-bin-2.3.8-r3.ebuild]: I'm gonna give the `doexe` a try and test this, just to learn a bit.

**Comment by natimerry** [comment]
>Maybe also give the doexe issue a try.

I did try that before making this PR, didnt work so I just fperms.  
>I will have a look later and test this out.

Just make sure the RPC launches with `/usr/lib/an-anime-game-launcher-bin/public/discord-rpc/discord-rpc -a 901534333360304168 -d "Bullying Paimon" -s "In the weeb game" -li "gi-icon" -st 1650682087`

**Comment by Schievel1** [comment]
Just changed that a little with the doexe, permissions are good now. Maybe you want to give this a try

**Comment by Schievel1** [comment]
```
❯ /usr/lib/an-anime-game-launcher-bin/public/discord-rpc/discord-rpc -a 901534333360304168 -d "Bullying Paimon" -s "In the weeb game" -li "gi-icon" -st 1650682087
/usr/lib/an-anime-game-launcher-bin/public/discord-rpc/discord-rpc
-a
901534333360304168
-d
Bullying Paimon
-s
In the weeb game
-li
gi-icon
-st
1650682087
[1]    30654 terminated  /usr/lib/an-anime-game-launcher-bin/public/discord-rpc/discord-rpc -a  -d  -s
```
When I run it i get this. So at least permissions are ok now. Dunno if this output is good or bad and certainly don't know if it is intended behavior. :D

**Comment by natimerry** [comment]
> ```
> ❯ /usr/lib/an-anime-game-launcher-bin/public/discord-rpc/discord-rpc -a 901534333360304168 -d "Bullying Paimon" -s "In the weeb game" -li "gi-icon" -st 1650682087
> /usr/lib/an-anime-game-launcher-bin/public/discord-rpc/discord-rpc
> -a
> 901534333360304168
> -d
> Bullying Paimon
> -s
> In the weeb game
> -li
> gi-icon
> -st
> 1650682087
> [1]    30654 terminated  /usr/lib/an-anime-game-launcher-bin/public/discord-rpc/discord-rpc -a  -d  -s
> ```
> 
> When I run it i get this. So at least permissions are ok now. Dunno if this output is good or bad and certainly don't know if it is intended behavior. :D

It is intended behaviour, I would say it is ready to be merged now, you can further test it by opening discord (make sure activity status is enabled) and running launcher with RPC checked.

**Comment by Schievel1** [comment]
Weeeeeeee there we go.....

Thanks again

**Comment by natimerry** [comment]
Thanks to you to, now to get Ltrinch to test it for me one last time.

**Pull request merged**